cmd/delegation/main.go

```diff
@@ -13,26 +13,44 @@ import (
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	"github.com/interchainio/delegation/pkg"
 	tmclient "github.com/tendermint/tendermint/rpc/client"
+
+	"github.com/spf13/cobra"
 )
 
 var (
 	cdc = gaia.MakeCodec()
 
-	// expects a locally running node
-	node = tmclient.NewHTTP("localhost:26657", "/websocket")
-
-	// expects the gos.json file to be here
-	gosJSON = "data/gos.json"
-
-	// output unsigned delegation tx
-	outputFile = "unsigned-delegations.json"
+	fullNodeUrl string
```

Comment:
Rather use the fully capitalized version of the variable name: 
```suggestion
	fullNodeURL string
```